### PR TITLE
Update server to correct version.

### PR DESCRIPTION
**Pull request by terrylucas**
Noticed that the devtools_server's version should have been 0.1.13 then all dependencies should be to 0.1.13

### CODE REVIEW

File: packages/devtools_server/pubspec.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: devtools_server
 description: A server that supports Dart DevTools.
 
-version: 0.1.13-dev
+version: 0.1.13
```
Comment (kenzieschmoll) [inline packages/devtools_server/pubspec.yaml]: Were we depending on some unreleased feature in the server? Do we need to release devtools_server first?

Comment (terrylucas) [reply packages/devtools_server/pubspec.yaml]: I don't know that is my concern that devtools' dependency is today on 0.1.12.  So I'm okay with leaving it this way for 0.1.14 and we figure if we should migrate to 0.1.13 later.

Comment (kenzieschmoll) [reply packages/devtools_server/pubspec.yaml]: sg to leave as is for 0.1.14. I filed an [issue](https://github.com/flutter/devtools/issues/1598) to investigate this for the next release

Comment (terrylucas) [reply packages/devtools_server/pubspec.yaml]: The change we haven't really tested is the server using the latest sse package.

**Pull request closed without merging**